### DISCUSSION TITLE
天翼云添加后隔几天报错，如何避免

### DISCUSSION THREAD

**Discussion by OPPO9008** [AlistGo/alist#920] [Q&A] [answered]
userSessionBO is null or fail to get sessionsecret by sessionkey 

在账号编辑重新点击保存又好了
希望自动刷新

**Comment by gzxiexl** [comment]
我也遇到了

**Comment by anwen-anyi** [comment]
添加的账号是 189CloudPC还是 189Cloud？

**Reply by anwen-anyi** [reply]
> 添加的账号是 189CloudPC还是 189Cloud？

确实,刚准备去添加一个家庭云来着.发现我之前几天添加的个人云也失效了..emmm

**Comment by lewis1573** [comment]
189CloudPC，也遇到了这个问题

**Reply by anwen-anyi** [reply, marked as answer]
更新版本（最新2.5.0）,删掉账号,重新添加
